pkg/porter/outputs.go

```diff
@@ -38,30 +55,125 @@ func (l Outputs) Less(i, j int) bool {
 	return siLower < sjLower
 }
 
-func (p *Porter) listBundleOutputs(bundle string) (*Outputs, error) {
+// Validate validates the provided args, using the provided context,
+// setting attributes of OutputShowOptions as applicable
+func (o *OutputShowOptions) Validate(args []string, cxt *context.Context) error {
+	switch len(args) {
+	case 0:
+		return errors.New("an output name must be provided")
+	case 1:
+		o.Output = args[0]
+	default:
+		return errors.Errorf("only one positional argument may be specified, the output name, but multiple were received: %s", args)
+	}
+
+	// If not provided, attempt to derive claim name from context
+	if o.sharedOptions.Name == "" {
+		err := o.sharedOptions.defaultBundleFiles(cxt)
+		if err != nil {
+			return errors.New("claim name must be provided via [--claim|-c CLAIM]")
+		}
+	}
+
+	return nil
+}
+
+// Validate validates the provided args, using the provided context,
+// setting attributes of OutputListOptions as applicable
+func (o *OutputListOptions) Validate(args []string, cxt *context.Context) error {
+	// Ensure only one argument exists (claim name) if args length non-zero
+	err := o.sharedOptions.validateClaimName(args)
+	if err != nil {
+		return err
+	}
+
+	// If args length zero, attempt to derive claim name from context
+	err = o.sharedOptions.defaultBundleFiles(cxt)
```

Comment:
Based on the comment, I expected an if statement here.

Reply:
Good call; comment wording a bit confusing.  Updated.